airbyte-scheduler/persistence/src/test/java/io/airbyte/scheduler/persistence/DefaultJobCreatorTest.java

```diff
@@ -151,7 +151,9 @@ void testCreateSyncJob() throws IOException {
         STANDARD_SYNC,
         SOURCE_IMAGE_NAME,
         DESTINATION_IMAGE_NAME,
-        List.of(STANDARD_SYNC_OPERATION)).orElseThrow();
+        List.of(STANDARD_SYNC_OPERATION),
+        null,
```

Comment:
We should add some tests where these values are not null

Reply:
Actually I dont think tests with non-null values here would be useful, since the class just simply passes them through